cmd/importer/README.md

```diff
@@ -0,0 +1,42 @@
+# Kueue Importer Tool
+
+A tool able to import existing pods into kueue.
+
+## Cluster setup.
+
+The importer should run in a cluster having the Kueue CRDs defined and in which the `kueue-controller-manager` is not running or has the `pod` integration disabled.
+
+For an import to succeed, all the involved Kueue objects (LocalQueues, ClusterQueues and ResourceFlavors) need to be created in the cluster, the check stage of the importer will check this and enumerate the missing objects. 
+
+## Build
+
+From kueue source root run:
+ ```bash
+go build  -C cmd/importer/ -o $(pwd)/bin/importer
+
+ ```
+
+## Usage
+
+The command runs against the systems default kubectl configuration. Check the [kubectl documentation](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) to learn more about how to Configure Access to Multiple Clusters.
+
+The will perform following checks:
+
+- At least one `namespace` is provided.
+- The label key (`queuelabel`) providing the queue mapping is provided.
+- A mapping from ane of the encountered `queuelabel` values to an existing LocalQueue exists.
+- The LocalQueues involved in the import are using an existing ClusterQueue.
+- The ClusterQueues involved have at least one ResourceGroup using an existing ResourceFlavor. This ResourceFlavor is used when the importer creates the admission for the created workloads.
```

Comment:
Why at least one? What will happen if it uses non-existing ResourceFlavors? 
Probably we should at least log warning.

Reply:
If the first resource flavor is not created, the check will fail.